### DISCUSSION TITLE
Add support to select/deselect the failed jobs that needs to be re-run

### DISCUSSION THREAD

**Discussion by ashikkumar23** [community/community#67195] [Actions]
### Select Topic Area

Product Feedback

### Body

It would be nice to support, select or deselect the failed jobs that needs to be re-run.

![image](https://github.com/community/community/assets/63716186/cbf9f56f-97b3-4888-90e4-e49551019453)

Suppose, I am interested to run only the second failed job; currently it is not possible as it would run all the failed jobs (`A new attempt of this workflow will be started, including all failed jobs and dependents`)!

Providing an option to select and deselect would be a handy feature.

Thanks!

**Comment by kkscilife** [comment]
I also want to re-run just failed jobs, no dependents.
